### DISCUSSION TITLE
newpointblank

### DISCUSSION THREAD

**Discussion by orangemonkey01** [3kh0/ext-remover#667] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
what does the newpointblank do?

**Comment by BroJac5246** [comment, marked as answer]
New Point Blank is when an HTML injection bug is taken advantage of to allow us to open a page where we can run bookmarklets that affect a page that would otherwise be protected. Here's how it works:

1. Find an HTML injection bug on an extension page and insert a link to `about:blank` with `rel="opener"` and `target="_blank"`
2. Click the link you injected. Because of `rel="opener"`, Chrome will link the two pages and they will be able to affect one another.
3. Although the extension page is protected from bookmarklets, `about:blank` is not. Run a bookmarklet there, and it is capable of affecting the extension, which could include disabling it or forcing it to reset itself.

**Reply by Demonic-Roses** [reply]
Could you explain how to do this?

**Reply by Aka-but-nice** [reply]
:skull:

**Reply by BroJac5246** [reply]
> Could you explain how to do this?

Generally you would want to find someone who already made an exploit.

**Comment by Aka-but-nice** [comment]
> Could you explain how to do this?

What restricting-extension does your school-issued Chromebook use?
